aptos-move/aptos-vm/src/adapter_common.rs

```diff
@@ -53,6 +54,15 @@ pub(crate) trait VMAdapter {
         txn: &PreprocessedTransaction,
         data_cache: &impl MoveResolverExt,
         log_context: &AdapterLogSchema,
+        aggregator_enabled: bool,
+    ) -> Result<(VMStatus, VMOutput, Option<String>), VMStatus>;
+
+    /// Execute a single transaction when running the block sequentially.
```

Comment:
Can we add a comment describing why we have `execute_single_transaction_sequential` and `execute_single_transaction`, i.e. problems with materialization and aggregator_disabled flag? Just to make sure this is documented in the code :)

Reply:
Done.